cmd/state/internal/cmdtree/auth.go

```diff
@@ -52,15 +58,23 @@ func newAuthCommand(prime *primer.Values) *captain.Command {
 
 func newSignupCommand(prime *primer.Values) *captain.Command {
 	signupRunner := auth.NewSignup(prime)
+	interactive := false
```

Comment:
Please use the same `params` pattern as used with the `newAuthCommand` implementation.

Reply:
Done.